ui/shared/Revision.jsx

```diff
@@ -94,13 +127,24 @@ export class Revision extends React.PureComponent {
         </span>
         <AuthorInitials title={`${name}: ${email}`} author={name} />
         <span
-          title={comment}
+          ref={this.spanRef}
           className={`ml-2 revision-comment overflow-hidden text-nowrap ${commentColor}`}
+          id={`revision${revision}`}
         >
           <em>
-            <BugLinkify>{comment}</BugLinkify>
+            <BugLinkify id={revision}>{comment}</BugLinkify>
           </em>
         </span>
+        {ready && (
+          <Tooltip
+            isOpen={tooltipOpen}
+            toggle={() => this.handleTooltip(comment)}
+            target={`revision${revision}`}
```

Comment:
Here I would create a CSS class in `treeherder-pushes.css` and use it as the value for `innerClassName`.  The CSS class should left-align the text and set a `min-width` and `max-width: 100%`

Reply:
css class tooltip-content created